packages/eslint-plugin-typescript/src/configs/import.ts

```diff
@@ -3,6 +3,8 @@ import {TSESLint} from '@typescript-eslint/utils';
 const pluginPrefix = 'import-x';
 
 export const importEslintRules: TSESLint.Linter.RulesRecord = {
+    // computationally expensive and asterisk imports are not best practice anyway
```

Comment:
isn't the comment a bit missleading. Seems like this rule is useless with typescript as it won't compile anyway if some import doesn't exist

Reply:
Fixed.